openmc/universe.py

```diff
@@ -373,6 +379,22 @@ def plot(self, origin=(0., 0., 0.), width=(1., 1.), pixels=(200, 200),
         elif basis == 'xz':
             x, y = 0, 2
             xlabel, ylabel = 'x [cm]', 'z [cm]'
+
+        # checks to see if bounding box contains -inf or inf values
+        if True in np.isinf(self.bounding_box):
```

Comment:
Just realized this is overly constraining. Technically if the bounding box is infinite in only one dimension (e.g., along z axis), we can still automatically figure out the width in the other directions (for a 'xy' plot). I'll leave it up to you if you want to implement a check for this.

Reply:
I've raised an issue and assigned myself.